src/webhooks/test/registry.test.ts

```diff
@@ -443,15 +428,15 @@ describe('ShopifyWebhooks.Registry.process', () => {
   });
 
   afterEach(async () => {
-    ShopifyWebhooks.Registry.webhookRegistry = [];
+    ShopifyWebhooks.Registry.webhookRegistry = {};
   });
 
   it('handles the request when topic is already registered', async () => {
-    ShopifyWebhooks.Registry.webhookRegistry.push({
+    ShopifyWebhooks.Registry.webhookRegistry.PRODUCTS = {
```

Comment:
We should replace all of these calls that are directly setting values to `webhookRegistry` to call `addHandler` instead to test the method.

Reply:
Yeah I didn't know if I should do that because if the `addHandler` function breaks it would break the other tests. This makes more sense though because if the `addHandler` function breaks everything else would break anyways